vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/model/implementation/DataSourceDeclarationImpl.java

```diff
@@ -85,7 +85,7 @@ public boolean equals(final Object obj) {
 
 	@Override
 	public String toString() {
-		return "@source " + this.predicate.toString() + "(" + this.predicate.getArity() + ") : "
+		return "@source " + Serializer.getPredicateString(this.predicate) + "(" + this.predicate.getArity() + ") : "
```

Comment:
This does not correspond to our syntax.
`getPredicateString` returns also the arity of the predicate.
it should be
`"@source " + this.predicate.getName() + "(" + this.predicate.getArity() + ") : " + this.dataSource.toConfigString() + " .";`